cid/helpers/quicksight/__init__.py

```diff
@@ -238,13 +238,45 @@ def discover_dashboard(self, dashboardId: str) -> Dashboard:
             templateAccountId = _definition.get('sourceAccountId')
             templateId = _definition.get('templateId')
             region = _definition.get('region', 'us-east-1')
+
             try:
                 template = self.describe_template(templateId, account_id=templateAccountId, region=region)
                 dashboard.sourceTemplate = template
             except Exception as e:
                 logger.debug(e, exc_info=True)
                 logger.info(f'Unable to describe template {templateId} in {templateAccountId} ({region})')
 
+            # Checking for version override in template definition
+            for dashboard_template in [dashboard.deployedTemplate, dashboard.sourceTemplate]:
+                if isinstance(dashboard_template, CidQsTemplate):
```

Comment:
(minor) can be much shorter


```
```
  if not isinstance(dashboard_template, CidQsTemplate)\
     or not dashboard_template.version \
     or not _definition.get('versions'): 
     continue
```
                    version = 

                    if version > 0:
                        version_obj = _definition.get('versions')

                        if version_obj:
```

Reply:
Nicely done